starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/MainActivity.java

```diff
@@ -1,26 +1,47 @@
 package ly.generalassemb.drewmahrt.shoppinglistver2;
 
-import android.content.Context;
-import android.database.Cursor;
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.CursorAdapter;
-import android.widget.ListView;
-import android.widget.SimpleCursorAdapter;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import java.util.List;
 
 import ly.generalassemb.drewmahrt.shoppinglistver2.setup.DBAssetHelper;
 
 public class MainActivity extends AppCompatActivity {
+    RecyclerView mRecyclerView;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        //RECYCLERVIEW STUFF
+        mRecyclerView = (RecyclerView) findViewById(R.id.recyclerview);
+
+        //Create the Layout Manager:
+        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(MainActivity.this,
+                LinearLayoutManager.VERTICAL, false);
+
+        //Now that we created; call the layout manager
+        mRecyclerView.setLayoutManager(linearLayoutManager);
+
+        //Populate the list.
+        List<ItemObject> list = LabSQLiteOpenHelper.getInstance(this).getAllAsList();
+
+        //Set the adapter.
+        mRecyclerView.setAdapter(new RecyclerAdapter(list));
+        //RECYCLERVIEW STUFF ENDS
+
         //Ignore the two lines below, they are for setup
         DBAssetHelper dbSetup = new DBAssetHelper(MainActivity.this);
         dbSetup.getReadableDatabase();
```

Comment:
These two lines, which populate the database with entries, need to be moved up before you query the database to populate your list for the RecyclerView